site/assets/scss/_syntax.scss

```diff
@@ -27,22 +27,23 @@
 // TODO: these values will need to be modified in order to work with Boosted dark mode
 // Right now, values are from Bootstrap from `--base00` to `--baseOF` and the 4 remaining one are copied from Boosted's light mode
 @include color-mode(dark, true) {
-  --base00: #282c34;
+  // --base00: #282c34;
+  // --base01: #353b45;
   --base01: #353b45;
```

Comment:
This one shouldn't appear uncommented right ?

Reply:
Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1842/commits/76f5c7288b39954dc1075157f50a03afc691a708